src/plugins/intel_gna/CMakeLists.txt

```diff
@@ -75,3 +75,7 @@ file(GLOB_RECURSE source_list "${libGNA_LIBRARIES_BASE_PATH}/*${CMAKE_SHARED_LIB
 install(FILES ${source_list}
         DESTINATION ${IE_CPACK_RUNTIME_PATH}
         COMPONENT gna)
+
+install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/cmake/libGNAConfig.cmake"
+        DESTINATION runtime/cmake
+        COMPONENT gna)
```

Comment:
should be installed only when we build static libraries (`BUILD_SHARED_LIBS=OFF`)

Reply:
done